easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -209,16 +210,26 @@ def configure_step(self, srcdir=None, builddir=None):
         options.append('-DCMAKE_FIND_USE_PACKAGE_REGISTRY=FALSE')
 
         if not self.cfg.get('allow_system_boost', False):
-            # don't pick up on system Boost if Boost is included as dependency
-            # - specify Boost location via -DBOOST_ROOT
-            # - instruct CMake to not search for Boost headers/libraries in other places
-            # - disable search for Boost CMake package configuration file
             boost_root = get_software_root('Boost')
             if boost_root:
+                boost_version = get_software_version('Boost')
+                # Check for older builds of Boost
+                cmake_files = glob.glob(os.path.join(boost_root, 'lib', 'cmake', 'boost_system-%s' % boost_version,
```

Comment:
You could use this to rely even less on the specific naming and avoid the need to get the version

```suggestion
                cmake_files = glob.glob(os.path.join(boost_root, 'lib', 'cmake', 'boost_system*',
```

Reply:
Yes, but i wanted to enforce the version there.

Reply:
I'm just a bit cautious here for the case that Boost did or will change the naming, because from the cmake-side you can omit the version in the name. I don't see a benefit in enforcing the version, the version is already encoded in the root module folder, so there can't be anything else here.
But up to you, as far as I know this folder has always included the version so this works now.

Reply:
You're right, it is a bit overkill...

Reply:
Fixed it.